tequilapi/endpoints/proposals.go

```diff
@@ -65,6 +65,19 @@ type proposalRes struct {
 
 	// qualitative service definition
 	ServiceDefinition serviceDefinitionRes `json:"serviceDefinition"`
+
+	// Quality of the service
+	Quality *serviceQuality `json:"quality,omitempty"`
```

Comment:
Sometime `null` simetimes structure `{}`?

Reply:
Changed it, now if `fetchQuality` flag is given, it prints full struct even with zero values.